extensions/src/ACRE2Core/FilterRadio.cpp

```diff
@@ -89,6 +94,7 @@ CFilterRadio::CFilterRadio(void)
     srand((unsigned int)time(0));
     this->m_HighPass.setup (TS_SAMPLE_RATE, 750, 0.97);
     this->m_LowPass.setup (TS_SAMPLE_RATE, 4000, 2.0);
+    this->m_lowShelf.setup(TS_SAMPLE_RATE, 1000, -20, 1);
```

Comment:
-20db matches the video, 
but I think it would be better to make the effect more subtle so we don't have to add a setting for this
probably 10-15

Reply:
I think -20db is fine honestly, but we can set it to -15db so it doesn't become "useless" on certain audio setups.

Reply:
Can it be a setting? Not sure how that would work for the extension

Reply:
I don't see a need for a setting. This is more realistic but not that different.